practical-react-unit-testing-with-jest-mocks-and-spies/Exercise1/problem/src/Select.test.tsx

```diff
@@ -0,0 +1,44 @@
+/**
+ * @jest-environment jsdom
+ */
+//manually mocking toSelectOptions
+
+import { render } from "@testing-library/react";
+import { Select } from "./Select";
+import { toSelectOptions } from "./toSelectOptions";
+
+// mock the toSelectOptions module with a callback that returns a mock module
+//   mocking the toSelectOptions function
+
+describe("Year", () => {
+  test("Year works with one-off mocked values", () => {
+    // Use mockImplementationOnce to get a 2024 value for just this one render
```

Comment:
I would try to make the exercise as clear as possible in the comments. You can do this by:

- Adding a comment to the top of the file that outlines each requirement: https://github.com/bitovi/trainings/blob/024e679bafa2a7275b253997b3036ad0412e7185/incorporating-accessibility-into-your-dev-process/forms/solution/src/App.tsx#L1-L11

- Adding inline comments (like this one) but making it very clear it’s part of the exercise, e.g. `// TODO` or `// Exercise 1)` or something like that)